pages/api/submit.ts

```diff
@@ -97,9 +97,9 @@ const callLambda = async (formID: string, fields: Responses, language: string) =
 };
 
 const previewNotify = async (form: PublicFormSchemaProperties, fields: Responses) => {
-  const templateID = "92096ac6-1cc5-40ae-9052-fffdb8439a90";
+  const templateID = process.env.TEMPLATE_ID;
   const notify = new NotifyClient(
-    "https://api.notification.canada.ca",
+    process.env.GC_NOTIFY_URL,
```

Comment:
This is not present in the ecs form_viewer task JSON which means it wont be present in staging and production. See comment on your PR in forms-terraform. I think going back to a hardcoded value is preferred since this is static and unchanging.